### PR TITLE
Risk score calculation for closed alerts

### PR DESCRIPTION
**Pull request by natasha-moore-elastic**
Resolves https://github.com/elastic/security-docs/issues/6254.

### Previews

ESS:
* [Entity risk scoring | How is risk score calculated](https://security-docs_bk_6271.docs-preview.app.elstc.co/guide/en/security/master/entity-risk-scoring.html#how-is-risk-score-calculated)
* [Turn on the risk scoring engine | Turn on the latest risk engine](https://security-docs_bk_6271.docs-preview.app.elstc.co/guide/en/security/master/turn-on-risk-engine.html#_turn_on_the_latest_risk_engine)

Serverless
* [Entity risk scoring | How is risk score calculated](https://security-docs_bk_6271.docs-preview.app.elstc.co/guide/en/serverless/main/security-entity-risk-scoring.html#security-entity-risk-scoring-how-is-risk-score-calculated)
* [Turn on risk scoring | Turn on the risk engine](https://security-docs_bk_6271.docs-preview.app.elstc.co/guide/en/serverless/main/security-turn-on-risk-engine.html#security-turn-on-risk-engine-turn-on-the-risk-engine)

### CODE REVIEW

**Comment by natasha-moore-elastic** [comment]
Currently blocked on updating screenshots due to no available environment.

**Review by benironside — APPROVED** [review]
I know we're waiting on updated screenshots, but the text LGTM

**Review by abhishekbhatia1710 — APPROVED** [review]

**Pull request merged**
